src/services/auth/plugins/magicLink/schemas.ts

```diff
@@ -1,72 +1,53 @@
+import { Type } from '@sinclair/typebox';
+
 import { MAX_USERNAME_LENGTH, MIN_USERNAME_LENGTH } from '@graasp/sdk';
 
 import { SHORT_TOKEN_PARAM } from '../passport';
 
 export const register = {
-  body: {
-    type: 'object',
-    required: ['name', 'email', 'captcha'],
-    properties: {
-      name: {
-        type: 'string',
+  body: Type.Object(
+    {
+      name: Type.String({
+        format: 'username',
         minLength: MIN_USERNAME_LENGTH,
         maxLength: MAX_USERNAME_LENGTH,
-        format: 'username',
-      },
-      email: { type: 'string', format: 'email' },
-      captcha: { type: 'string' },
-      url: {
-        type: 'string',
-        format: 'uri',
-      },
-      enableSaveActions: { type: 'boolean' },
+      }),
+      email: Type.String({ format: 'email' }),
+      captcha: Type.String(),
+      url: Type.Optional(Type.String({ format: 'uri' })),
```

Comment:
I know it was already like that, but should we rename the format by `url` instead of `uri`? As the property is an url and not an uri?

Reply:
I've made a task in the big refactor issue